### PR TITLE
Set Up Rails Application

### PR DESCRIPTION
**Pull request by JohnFTitor**
# Hi there 👋🏻

Thanks for taking the time to review the project. Here are some things important to note about it:

### Changes implemented in this PR

- [X] Solves #1 
- [X] Add a Pull Request Template to keep consistency during the development of the project

## Final notes 📔 

Is expected that this project meets the requirements. Please let me know if anything is left to be done or can be improved. Again, thanks for the review.

j

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
+# gem "image_processing", "~> 1.2"
+
+# Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
+# gem "rack-cors"
+
+group :development, :test do
+  # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
+  gem 'debug', platforms: %i[mri mingw x64_mingw]
+  gem 'factory_bot'
```
Comment (jose-Abel) [inline Gemfile:42]: Sorry that it wasn't specify correctly in the task, but this is not the gem to create factories with Rails, searching in google this gem (factory_bot), it'll show the Github repo for this gem and on that repo indicates: "If you want to use factory_bot with Rails, see [factory_bot_rails](https://github.com/thoughtbot/factory_bot_rails)." So the right gem to use factory_bot in Rails is factory_bot_rails. Please fix this and it'll be approved! :)

**Review by jose-Abel — CHANGES_REQUESTED** [review]
# Hi @JohnFTitor 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
## Required Changes :recycle:
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is just a personal suggestion that I think can make your code better. I'm open to discussing your opinions around them but are not required so we can continue the development of the project.
You can also consider:
- x
- y
- z
Cheers and Happy coding!:clap::clap::clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
------
This is a feature teammate review. Remember that everything said here is intended for both of us to learn and grow as Software Developers :computer:  and we may have different approaches or perspectives. That being said, don't hesitate to comment if you disagree with one required change. We can talk everything through :cat:

File: Gemfile (hunk 1 above)
Comment (JohnFTitor) [reply Gemfile]: You're totally right. I miss it.
It's fixed now. Thanks for pointing it out!

**Review by jose-Abel — APPROVED** [review]
# Hi @JohnFTitor ,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:
## Highlights
## Optional suggestions
Every comment with the [OPTIONAL] prefix is just a personal suggestion that I think can make your code better. I'm open to discussing your opinions around them but are not required so we can continue the development of the project.
Cheers and Happy coding!:clap::clap::clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
------
This is a feature teammate review. Remember that everything said here is intended for both of us to learn and grow as Software Developers :computer:  and we may have different approaches or perspectives. That being said, don't hesitate to comment if you disagree with one required change. We can talk everything through

**Pull request merged**